stix2elevator/test/test_idioms.py

```diff
@@ -35,7 +35,7 @@ def idiom_mappings(xml_file_path, stored_json):
     if not get_option_value("policy") == "no_policy":
         print("'no_policy' is not allowed for testing")
     set_option_value("policy", "no_policy")
-
+    sys.setrecursionlimit(5000)
```

Comment:
This is already defined in `setup_tests` (recently merged into master)

Reply:
removed

Reply:
Did you push these changes to GitHub? I'm still seeing this line in the pattern-expression-classes branch